code/game/objects/effects/anomalies.dm

```diff
@@ -364,12 +364,12 @@
 	icon = 'icons/obj/projectiles.dmi'
 	icon_state = "radiation_anomaly"
 	density = TRUE
-	var/has_effect = TRUE //For goat spawning
+	var/spawn_goat = TRUE //For goat spawning
 
 /obj/effect/anomaly/radiation/anomalyEffect()
 	..()
-	for(var/i = 1 to 10)
-		fire_nuclear_particle_wimpy()
+	for(var/i = 1 to 15)
+		fire_nuclear_particle()
```

Comment:
not only making them the stronger particles, but firing 5 more?
that's a pretty significant increase

Reply:
well i want the radiation anomaly to be lethal and not just a small orb shooting tickle particles that barely harming mob. 
I want the crew to actually avoiding it, and force them to use a radiation suit when going near it